Level_1/pick_game.py

```diff
@@ -0,0 +1,27 @@
+def solution(board, moves):
+    answer = 0
+    arr = []
+    bucket = []
+
+    # column list 생성
+    for i in range(len(board[0])):
+        line = []
+        for j in range(len(board)):
+            if board[j][i] != 0:
+                line.append(board[j][i])
+        arr.append(line)
+
+    # moves 바구니 실행
+    for i in moves:
+        if len(arr[i-1]) and bucket:
```

Comment:
`bucket` 과 동일하게 `arr[i-1]` 도 해주시면 될듯?
```suggestion
        if arr[i-1] and bucket:
```

그리고 bucket 에 0 같이 의미없는 숫자 하나 넣어놓고 시작하면 
비어있는지 검사를 안해도 될거같고, 코드량을 많이 줄일 수 있을 듯 하네용,

Reply:
넵 참고해서 수정해 보도록 할게요